packages/venia-concept/src/components/MiniCart/totalsSummary.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { number, shape, string } from 'prop-types';
+
+import { Price } from '@magento/peregrine';
+
+import { mergeClasses } from 'src/classify';
+
+import defaultClasses from './totalsSummary.css';
+
+const TotalsSummary = props => {
+    // Props.
+    const { currencyCode, numItems, subtotal } = props;
+
+    // Members.
+    const classes = mergeClasses(defaultClasses, props.classes);
+    const hasSubtotal = Boolean(subtotal);
+    const numItemsText = numItems === 1 ? 'item' : 'items';
+
+    return (
+        <div className={classes.root}>
+            {hasSubtotal && (
+                <dl className={classes.totals}>
+                    <dt className={classes.subtotalLabel}>
+                        <span>
+                            Cart Total :&nbsp;
```

Comment:
```suggestion
                            {'Cart Total : '}
```

Reply:
Resolved in https://github.com/magento-research/pwa-studio/pull/1266/commits/2f4a5ad5a131e766b7064575ef2b5b72cf3a3d9e.